presto-main/src/main/java/com/facebook/presto/execution/scheduler/NodeScheduler.java

```diff
@@ -68,19 +68,21 @@
     private final int maxPendingSplitsPerTask;
     private final NodeTaskMap nodeTaskMap;
     private final boolean useNetworkTopology;
+    private final long nodeMapRefreshInNanos;
```

Comment:
Please use `io.airlift.units.Duration` instead.

See the 9th bullet point here: https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development